fs/_bulk.py

```diff
@@ -88,7 +97,7 @@ def __init__(self, num_workers=4):
     def start(self):
         """Start the workers."""
         if self.num_workers:
-            self.queue = Queue(maxsize=self.num_workers)
+            self.queue = Queue()
```

Comment:
Why the change here?

Reply:
see my other comment [here](https://github.com/PyFilesystem/pyfilesystem2/pull/463#discussion_r609513389)